docs/data/product/daily-satpaths-beta/_data.yaml

```diff
@@ -3,20 +3,23 @@
 
 # Overview
 
-title: Daily SatPaths
-long_title: Daily Satellite Paths predictor
+short_name: Daily SatPaths
+full_technical_name: Daily Satellite Paths predictor
 header_image: /_files/cmi/SatPaths_image.png
 version: 1.0.3-beta
 is_latest_version: true
 latest_version_link: null
 is_provisional: true
-product_type: Data service
-spatial_data_type: null
+type:
+  product_type: Data service
+  spatial_data_type: null
```

Comment:
@robbibt Would you know if this is a Raster or Vector product (or neither)?

Reply:
These are services, not data. SatPaths allows you to view the paths of the various satellites we use in DEA to work out when they will be flying overhead.

Reply:
In that case, I'll leave it as `Data service`